### ISSUE TITLE
VAX8600: Massbus changes result in MACHINECHK

### ISSUE THREAD

**Issue by rscottbailey** [simh/simh#307]
I'm pretty sure something in commits 820d77e..550e360 has upset VMS V7.3 running on my virtual VAX 8650. I just did a fresh rebuild and now, relatively early in the boot process, I get:

MACHINECHK, Machine check while in kernel mode

I have saved a copy of the dump in case you are so desperate as to want to look at it. ;-)

Possibly I have found my own problem after diff'ing the output of "show devices" before and after:

-sbailey@buzz:/vax$ ./vax8600.40-20160427 vax8600-diag.ini
+sbailey@buzz:/vax$ ./vax8600.40-20160509 vax8600-diag.ini

-VAX 8600 simulator V4.0-0 Beta        git commit id: 9ab41509
+VAX 8600 simulator V4.0-0 Beta        git commit id: cd032376
 RQ3: unit is read only
 vax8600-diag.ini-19> set tq0 tu81
 Invalid argument
@@ -14,7 +14,7 @@
 SBIA
 UBA     nexus=3, address=20006000, autoconfiguration enabled
 MBA0    disabled
-MBA1    disabled
+MBA1    nexus=9, address=20012000
 TODR
 TMR
 TTI     4 units

This led to the apparently-not-quite-correct thought:

sim> set mba1 disable
Command not allowed
sim>

Here's the entire configuration file I'm using, if it helps:

set cpu simhalt
set cpu 64m
set cr disable
set dz lines=8
set lpt disable
set rl disable
set rp disable
set ry disable
set hk disable
set ts disable
set tu disable
set rq0 rauser=4000
attach rq0 /vax/oasis.dsk
set rq1 rauser=4000
attach rq1 /vax/oasis2.dsk
set rq3 rrd40
attach -r rq3 /vax/vmsvax73.iso
set tq enable
set tq0 tu81
set tq0 capac=0
attach tq0 /vax/vax8600.tap
set tq1 disable
set tq2 disable
set tq3 disable
set xu enable
set xu type=delua
attach xu tap:tap0
set cpu idle=vms
set cpu model=8650
show device

Thanks,
-Scott

**Comment by markpizz** [comment]
Well:

    -MBA1 disabled
    +MBA1 nexus=9, address=20012000

By itself, should not be a problem.  Your configuration explicitly disables both RP and TU devices so there are no Massbus devices so I believe this should be fine (it is the behavior I intended when I added the 'auto configuration' support to the Massbus devices).  This problem is certainly reproducible in various ways on each of the VAX8600, VAX780 and VAX750.

I'm exploring what is actually happening.

- Mark
